src/Stache/Query/Builder.php

```diff
@@ -21,15 +22,42 @@ public function count()
         return $this->getFilteredAndLimitedKeys()->count();
     }
 
-    public function get($columns = ['*'])
+    private function resolveKeys()
     {
         $keys = $this->getFilteredKeys();
 
         $keys = $this->orderKeys($keys);
 
-        $keys = $this->limitKeys($keys);
+        return $this->limitKeys($keys);
+    }
+
+    public function pluck($column, $key = null)
+    {
+        $keys = $this->resolveKeys();
+
+        return $this->store->getFromIndex(
+            $this->getKeysForIndexQuery($keys),
+            $column,
+            $key
+        );
+    }
+
+    private function getKeysForIndexQuery($keys)
+    {
+        return $keys->map(function ($key) {
+            $queryKey = Str::after($key, '::');
+
+            if (! Str::contains($queryKey, '-') && is_numeric($queryKey)) {
+                return intval($queryKey);
+            }
```

Comment:
What is the purpose of this? Everything passes without it. Changing a numeric ID to an actual integer. But why?

Reply:
I'll have to try and re-create the scenario that caused issues on an actual site while working on this

Reply:
I could not for the life of me reproduce the setup where this was needed. I've removed it for now and will see if it ever pops up again